embulk-input-postgresql/src/main/java/org/embulk/input/PostgreSQLInputPlugin.java

```diff
@@ -49,11 +51,36 @@
         @ConfigDefault("false")
         public boolean getSsl();
 
+        @Config("ssl_version")
+        @ConfigDefault("null")
+        public Optional<SslVersion> getSslVersion();
```

Comment:
I changed implementation to accept dot separated value c27ebe2
Also updated README.md 7c78da1